packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -488,6 +488,11 @@ class RenderParagraph extends RenderBox {
         rect = rect.expandToInclude(textBox.toRect());
         currentDirection = textBox.direction;
       }
+      
+      if (rect == null) {
```

Comment:
This seems like a bug to me.  @jonahwilliams would know better, but I suspect we should not end up with a null rect here.